proposals/csharp-12.0/collection-expressions.md

```diff
@@ -205,6 +205,7 @@ ImmutableArray<int> ia =
 ```
 
 ## Construction
+[constructible-collection-types]: #constructible-collection-types
```

Comment:
Let's use <code>[construction]: #construction</code> here instead, and there shouldn't be any references to this section currently.